### ISSUE TITLE
Add a configurable title for the swagger ui

### ISSUE THREAD

**Issue by p013570** [gchq/Gaffer#1422]
This will help to brand the REST API Swagger UI for different systems.
This could include the graphId.
This should probably be done after #1420.

To demonstrate this, we should update the example projects (road-traffic, basic, federated) to have different tittles.

**Comment by p013570** [comment]
Merged into develop.
